mantidproject/fitbenchmarking
# Reorganised Code and Revamped Documentation

#### Description of Work

Fixes #141 #136 #125 

Reorganised multiple bits of code and added documentation accordingly.
Automatic tests will be broken in this PR, but will get fixed in the next.

#### Testing Instructions

1. Run the `example_runScripts.py` file with `mantid` and `scipy` as software.
2. In both cases, check if the results are as expected (i.e. tables and support pages are produced as in the last version of the code).
